### ISSUE TITLE
Close Albanylcm Google Group

### ISSUE THREAD

**Issue by gahansen** [SNLComputation/Albany#414]
Since the beginnings of the Albany project, we have maintained a Google Group for the project:

https://groups.google.com/forum/#!forum/albanylcm

I set up this group to recieve commit messages - that is most, if not all, the traffic that one sees on the forum. From time to time other messages have been sent there.

Given today's move of Albany to the SNLComputation organization - I'd like to propose that we also revisit this Google Group. Does anyone see value in continuing this group or migrating it to serve another purpose? If not, I'd propose that I delete it.

In summary - does anyone see value in continuing the Albanylcm Google Group?

**Comment by ikalash** [comment]
I’m ok closing it except accesss is managed thrift it, no?

**Comment by ikalash** [comment]
Was there an issue or notice about the move to the group as we discussed yesterday?  Probably some announcement should go out to notify all users.

**Comment by gahansen** [comment]
I don't think it controls access to anything, but sending an email to:

albanylcm@googlegroups.com

would email everyone on that group that have been seeing Albany commit messages in their inbox. If group members have turned off email notifications, they would not see such an email.

How many Albany users see commit emails coming in to their inboxes from

albanylcm@googlegroups.com ???

**Comment by gahansen** [comment]
We could notify everyone of Albany's organization change to

SNLComputation/Albany

using this list. But I suspect most have turned off or redirected such emails and thus would not see such an email. I'll post an issue about the move now.

**Comment by ikalash** [comment]
Can this issue be closed?

**Comment by gahansen** [comment]
Lets leave it open a little longer - I am going to ping the list and make sure no one would be affected by its deletion.

**Comment by ikalash** [comment]
Ok, sure.

**Comment by gahansen** [comment]
I just sent a ping to the list:

- albanylcm@googlegroups.com

 
concerning deactivation of the list and supporting group. I did get an echo back - presumably anyone that is not filtering that list will see that email and will reply if they have objections to closing the list and group.

**Comment by gahansen** [comment]
I do note that GitHub has not yet deactivated the commit messages service, I just recieved Irina's commit message 62b292 (Feb 12, 2019 at 4:24PM) on the list.

**Comment by bartgol** [comment]
@gahansen any progress with this? Can we close? The list of Albany issues is getting a bit too long (we crossed 100 open issues), so I was hoping to close some stale ones...

**Comment by gahansen** [comment]
No comments or objections were received when I pinged the list. I'll delete the group now and close this issue.
